modelmatrix-core/src/main/scala/com/collective/modelmatrix/db/DatabaseConfig.scala

```diff
@@ -0,0 +1,56 @@
+package com.collective.modelmatrix.db
+
+import com.typesafe.config.{Config, ConfigFactory}
+import slick.driver.{H2Driver, JdbcProfile, PostgresDriver}
+
+/**
+ * Created by jpocalan on 7/27/15.
+ */
+// TODO: replace by some sort of enum
+// Differentiate between the different types of databse supported
+class DatabaseConfig(n: String, dc: String, sd: JdbcProfile) {
+  val name = n;
+  val driverClass = dc;
+  val slickDriver = sd;
+}
```

Comment:
This can be replaced with scala case class DatabaseConfig(name, driverClass, slickDriver) or just add val into parameter list to avoid re-defining variables inside class